packages/forklift-console-plugin/src/modules/Plans/views/details/components/SettingsSection/components/SharedDisksDetailsItem/SharedDisksDetailsItem.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+import { isPlanEditable } from 'src/modules/Plans/utils';
+import { useModal } from 'src/modules/Providers/modals';
+import { DetailsItem } from 'src/modules/Providers/utils';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { PlanDetailsItemProps } from '../../../DetailsSection';
+
+import MigrateSharedDisksModal from './components/MigrateSharedDisksModal/MigrateSharedDisksModal';
+import { getMigrateSharedDisks } from './utils/helpers';
+
+const SharedDisksDetailsItem: FC<PlanDetailsItemProps> = ({ resource, canPatch }) => {
+  const { t } = useForkliftTranslation();
+  const { showModal } = useModal();
+
+  const content = getMigrateSharedDisks(resource)
+    ? t('Migrate shared disks again')
+    : t('Migrate shared disks only once');
+
+  return (
+    <DetailsItem
+      title={t('Shared disks')}
+      content={content}
+      crumbs={['spec', 'migrateSharedDisks']}
+      onEdit={
+        canPatch &&
+        isPlanEditable(resource) &&
+        (() => showModal(<MigrateSharedDisksModal resource={resource} label={''} />))
```

Comment:
()=> {
if (canPatch && isPlanEdit) {
showModal(...)
}
}

Reply:
@metalice adding this change caused the edit pancil icon to show even though it's non functional:
![migrate-disks-pencil](https://github.com/user-attachments/assets/2d3b4419-7e03-4a86-97ab-7115ad545ff2)

will revert this change in next PR